battlePy/series.py

```diff
@@ -55,6 +56,14 @@ def __init__(self,
 
         self.player1.currentGame = self.games[0]
         self.player2.currentGame = self.games[0]
+        self.player1.totalTurns = 0
+        self.player1.averageTurns = 0
+        self.player1.minTurns = 1000
```

Comment:
Technically, it's possible a game could take longer that 1000 turns. I think it would be better to initialize these as null values.